contracts/strategy/anchor/AnchorNonUSTStrategy.sol

```diff
@@ -133,10 +133,17 @@ contract AnchorNonUSTStrategy is AnchorBaseStrategy {
      *
      * @notice since EthAnchor uses an asynchronous model, this function
      * only starts the deposit process, but does not finish it.
+     *
+     * @param data slippage information
      */
-    function invest() external override(AnchorBaseStrategy) onlyManager {
+    function invest(bytes calldata data)
+        external
+        override(AnchorBaseStrategy)
+        onlyManager
+    {
         require(initialized, "AnchorNonUSTStrategy: not initialized");
-        uint256 underlyingAmount = _swapUnderlyingToUst();
+        uint256 minAmount = abi.decode(data, (uint256));
```

Comment:
so `bytes data` needs to be set by us (backend?) to specify slippage, correct?

But this seems weird. If the backend sends the `invest(...)` transaction, and someone else is depositing at the same time, this slippage value will be outdated
shouldn't it be a min exchange rate instead of a raw amount?

Reply:
maybe, using min exchange rate could be reasonable.

Reply:
fixed